// random-in/dist/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const numbers_1 = require("./src/numbers");
const numbers_2 = require("./src/numbers");
const numbers_3 = require("./src/numbers");
const numbers_4 = require("./src/numbers");
const names_1 = require("./src/names");
const date_1 = require("./src/date");
const passwords_1 = require("./src/passwords");
const countries_1 = require("./src/countries");
const email_1 = require("./src/email");
const color_1 = require("./src/color");
const programming_1 = require("./src/programming");
module.exports.getNDigit = (n) => {
    return numbers_1.getNDigit(n);
};
module.exports.getRandomBetween = (min, max) => {
    return numbers_2.getRandomBetween(min, max);
};
module.exports.getRandomFloatBetween = (min, max) => {
    return numbers_3.getRandomFloatBetween(min, max);
};
module.exports.getNRandomBetween = (n, min, max) => {
    return numbers_4.getNRandomBetween(n, min, max);
};
module.exports.getName = () => {
    return names_1.getName();
};
module.exports.getMaleName = () => {
    return names_1.getMaleName();
};
module.exports.getFemaleName = () => {
    return names_1.getFemaleName();
};
module.exports.getMaleFirstName = () => {
    return names_1.getMaleFirstName();
};
module.exports.getFemaleFirstName = () => {
    return names_1.getFemaleFirstName();
};
module.exports.getLastName = () => {
    return names_1.getLastName();
};
module.exports.getIndianName = () => {
    return names_1.getIndianName();
};
module.exports.getIndianMaleName = () => {
    return names_1.getIndianMaleName();
};
module.exports.getIndianFemaleName = () => {
    return names_1.getIndianFemaleName();
};
module.exports.getIndianMaleFirstName = () => {
    return names_1.getIndianMaleFirstName();
};
module.exports.getIndianFemaleFirstName = () => {
    return names_1.getIndianFemaleFirstName();
};
module.exports.getIndianLastName = () => {
    return names_1.getIndianLastName();
};
module.exports.getDateBetween = (start, end) => {
    return date_1.getDateBetween(start, end);
};
module.exports.getRandomDateFuture = (futureYear) => {
    return date_1.getRandomDateFuture(futureYear);
};
module.exports.getAlphanumericPassword = (n) => {
    return passwords_1.getAlphanumericPassword(n);
};
module.exports.getLowercasePassword = (n) => {
    return passwords_1.getLowercasePassword(n);
};
module.exports.getUppercasePassword = (n) => {
    return passwords_1.getUppercasePassword(n);
};
module.exports.getAlphaPassword = (n) => {
    return passwords_1.getAlphaPassword(n);
};
module.exports.getAlphaLowercasePassword = (n) => {
    return passwords_1.getAlphaLowercasePassword(n);
};
module.exports.getAlphaUppercasePassword = (n) => {
    return passwords_1.getAlphaUppercasePassword(n);
};
module.exports.getNumericPassword = (n) => {
    return passwords_1.getNumericPassword(n);
};
module.exports.getCountry = () => {
    return countries_1.getCountry();
};
module.exports.getRandomEmail = () => {
    return email_1.getRandomEmail();
};
module.exports.getRandomColorHex = () => {
    return color_1.getRandomColorHex();
};
module.exports.getRandomProgrammingLanguage = () => {
    return programming_1.getRandomProgrammingLanguage();
};
