// random-in/dist/src/names.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const names_1 = require("../utils/names");
exports.getName = () => {
    let i = Math.floor(Math.random() * names_1.names.length);
    return names_1.names[i];
};
exports.getMaleName = () => {
    let i = Math.floor(Math.random() * names_1.maleNames.length);
    return names_1.maleNames[i];
};
exports.getFemaleName = () => {
    let i = Math.floor(Math.random() * names_1.femaleNames.length);
    return names_1.femaleNames[i];
};
exports.getMaleFirstName = () => {
    let maleFirstName = exports.getMaleName().split(' ')[0];
    return maleFirstName;
};
exports.getFemaleFirstName = () => {
    let femaleFirstName = exports.getFemaleName().split(' ')[0];
    return femaleFirstName;
};
exports.getLastName = () => {
    let lastName = exports.getName().split(' ')[1];
    return lastName;
};
exports.getIndianName = () => {
    let isMale = Boolean(Math.round(Math.random()));
    if (isMale) {
        return exports.getIndianMaleName();
    }
    else {
        return exports.getIndianFemaleName();
    }
};
exports.getIndianMaleName = () => {
    return `${exports.getIndianMaleFirstName()} ${exports.getIndianLastName()}`;
};
exports.getIndianFemaleName = () => {
    return `${exports.getIndianFemaleFirstName()} ${exports.getIndianLastName()}`;
};
exports.getIndianMaleFirstName = () => {
    let i = Math.floor(Math.random() * names_1.indianMaleFirstNames.length);
    return names_1.indianMaleFirstNames[i];
};
exports.getIndianFemaleFirstName = () => {
    let i = Math.floor(Math.random() * names_1.indianFemaleFirstNames.length);
    return names_1.indianFemaleFirstNames[i];
};
exports.getIndianLastName = () => {
    let i = Math.floor(Math.random() * names_1.indianLastNames.length);
    return names_1.indianLastNames[i];
};
