// random-in/dist/src/passwords.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.getAlphaPassword = (n) => {
    let charset = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
    let x = charset.length;
    let pass = '';
    for (let i = 0; i < n; ++i) {
        pass += charset.charAt(Math.floor(Math.random() * x));
    }
    return pass;
};
exports.getAlphaLowercasePassword = (n) => {
    let charset = 'abcdefghijklmnopqrstuvwxyz';
    let x = charset.length;
    let pass = '';
    for (let i = 0; i < n; ++i) {
        pass += charset.charAt(Math.floor(Math.random() * x));
    }
    return pass;
};
exports.getAlphaUppercasePassword = (n) => {
    let charset = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
    let x = charset.length;
    let pass = '';
    for (let i = 0; i < n; ++i) {
        pass += charset.charAt(Math.floor(Math.random() * x));
    }
    return pass;
};
exports.getNumericPassword = (n) => {
    let charset = '0123456789';
    let x = charset.length;
    let pass = '';
    for (let i = 0; i < n; ++i) {
        pass += charset.charAt(Math.floor(Math.random() * x));
    }
    return pass;
};
exports.getAlphanumericPassword = (n) => {
    let charset = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
    let x = charset.length;
    let pass = '';
    for (let i = 0; i < n; ++i) {
        pass += charset.charAt(Math.floor(Math.random() * x));
    }
    return pass;
};
exports.getLowercasePassword = (n) => {
    let charset = 'abcdefghijklmnopqrstuvwxyz0123456789';
    let x = charset.length;
    let pass = '';
    for (let i = 0; i < n; ++i) {
        pass += charset.charAt(Math.floor(Math.random() * x));
    }
    return pass;
};
exports.getUppercasePassword = (n) => {
    let charset = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
    let x = charset.length;
    let pass = '';
    for (let i = 0; i < n; ++i) {
        pass += charset.charAt(Math.floor(Math.random() * x));
    }
    return pass;
};
